events/class_tests/planner_tests.py

```diff
@@ -1,49 +1,121 @@
 import pytest
+from django.core import mail
+from users.models import User
+from datetime import timedelta
+from django.conf import settings
+from django.utils import timezone
+from django.forms import formset_factory
 from events.planner import EventPlanner
-from events.models import OptionalMeetingDates, PossibleParticipant, Event
+from django.contrib.messages import get_messages
+from events.class_models.participant_model import EventParticipant
+from events.models import OptionalMeetingDates, PossibleParticipant, Event, Colors
```

Comment:
Can you marge this lines into one ,like this
```suggestion
from events.models import OptionalMeetingDates, PossibleParticipant, Event, Colors, EventParticipant
```

Reply:
Done, thanks 😄